x/emissions/module/rewards/scores_test.go

```diff
@@ -320,7 +319,7 @@ func PrepareMockLosses(reputersCount int, workersCount int) (
 	reputersOneInNaiveLosses [][]alloraMath.Dec,
 ) {
 	for i := 0; i < reputersCount; i++ {
-		rand.Seed(time.Now().UnixNano())
```

Comment:
oh lord almighty

Reply:
why is this inside the for loop anyways, can't it just be seeded once

Reply:
yeah true